pkg/aggregator/demultiplexer.go

```diff
@@ -684,6 +673,67 @@ func (d *AgentDemultiplexer) GetMetricSamplePool() *metrics.MetricSamplePool {
 	return d.statsd.metricSamplePool
 }
 
+// GetRoutinesDistribution returns how many routines should be spawned for the
+// DogStatsD pipeline and how many DogStatsD workers should be running.
```

Comment:
The first value returned is the number of workers.

Reply:
Good catch.